docs/api.rst

```diff
@@ -47,6 +47,9 @@ Quickstart
 Valid search query keywords can be found at the `Copernicus Open Access Hub documentation
 <https://scihub.copernicus.eu/userguide/3FullTextSearch>`_.
 
+Search areas (i.e. ``map.geojson`` ) are provided as GeoJSON polygons, which can be created with
```

Comment:
No, maybe we should clarify this differently here: The `SentinelAPI.query` function actually takes the area of interest as WKT, not GeoJSON. 

The workflow `GeoJSON file > read to string > convert to WKT > pass to api.query` is pretty clear in this line `footprint = geojson_to_wkt(read_geojson('map.geojson'))`. 

So the primary thing to clarify here is that `map.geojson` is a path to a file named that, which was the source of issue #227.